HPXMLtoOpenStudio/tests/test_hvac_sizing.rb

```diff
@@ -336,6 +336,32 @@ def test_manual_j_residences
     assert_in_delta(998, hpxml_bldg.hvac_plant.cdl_lat_infil, default_tol_btuh)
     assert_in_delta(0, hpxml_bldg.hvac_plant.cdl_lat_vent, default_tol_btuh)
     assert_in_delta(1200, hpxml_bldg.hvac_plant.cdl_lat_intgains, default_tol_btuh)
+    # reduce tolerance for each space
+    default_tol_btuh_space = 50
+    hpxml_bldg.get_conditioned_zone.spaces.each do |space|
+      if space.id.include? 'dining'
+        assert_in_delta(902 + 407, space.hdl_walls, default_tol_btuh_space)
+        assert_in_delta(463, space.hdl_ceilings, default_tol_btuh_space)
+        assert_in_delta(779, space.hdl_infil, default_tol_btuh_space)
+        # assert_in_delta(153 + 69, space.cdl_sens_walls, default_tol_btuh_space) # Area correct, cooling htm in OS-HPXML: 0.51, Manual J: 1.23; Need further investigation
```

Comment:
@shorowit It turns out that the tests are passing with 50 btuh tolerance, probably I should add a relative error instead of absolute since the values do vary from 50 to over 1000. 
Two items that I want some attention for now:
1. Cooling wall loads being reported for WallDiningRoomNorth are much smaller than in the manual J, areas correct but the cooling HTM in our calculations are much smaller.
2.  Infiltration shows some diffs, about 50 btuh. I manually checked the wall area ratios, they're in great agreement. So it's the block loads that have some diffs.

Reply:
That's great that we are so close!

1. I looked into WallDiningRoomNorth a bit. In Section A12-13, it says "Table 4B provides wall CLTD values for walls. There are two CLTD values for each group number -- one for exposed walls, regardless of the direction the wall faces and one for partitions." But Figure A12-8 has different CLTDs for north walls vs non-directional walls. And Table 4B Footnote 1 says "Use the North CLTD values for externally shaded walls". I'm tempted to say we are doing it right and their value in the Long example is imperfect. I suggest allowing a larger tolerance for this wall and adding a comment explaining why.

2. I think the difference is because they use the Default Infiltration Table method (Table 5A) to get heating/cooling infiltration values, but we don't allow an input like "average" or "tight" in HPXML, we require an air infiltration measurement value and use the Blower Door Method. I manually calibrated the air infiltration measurement value to give similar results, but it gets heating a bit high and cooling a bit low (or vice versa, I forget). So again, I think we can use a slightly higher tolerance and add a comment. This reminds me that I think we should consider adding qualitative infiltration inputs (average, tight) to OpenStudio-HPXML. We already have an implementation in HEScore, so we could move it here. I'm going to add an issue around this, maybe you can work on that next.

Reply:
Issue added: https://github.com/NREL/OpenStudio-HPXML/issues/1652

Reply:
> That's great that we are so close!
> 
> 1. I looked into WallDiningRoomNorth a bit. In Section A12-13, it says "Table 4B provides wall CLTD values for walls. There are two CLTD values for each group number -- one for exposed walls, regardless of the direction the wall faces and one for partitions." But Figure A12-8 has different CLTDs for north walls vs non-directional walls. And Table 4B Footnote 1 says "Use the North CLTD values for externally shaded walls". I'm tempted to say we are doing it right and their value in the Long example is imperfect. I suggest allowing a larger tolerance for this wall and adding a comment explaining why.
> 2. I think the difference is because they use the Default Infiltration Table method (Table 5A) to get heating/cooling infiltration values, but we don't allow an input like "average" or "tight" in HPXML, we require an air infiltration measurement value and use the Blower Door Method. I manually calibrated the air infiltration measurement value to give similar results, but it gets heating a bit high and cooling a bit low (or vice versa, I forget). So again, I think we can use a slightly higher tolerance and add a comment. This reminds me that I think we should consider adding qualitative infiltration inputs (average, tight) to OpenStudio-HPXML. We already have an implementation in HEScore, so we could move it here. I'm going to add an issue around this, maybe you can work on that next.

Thanks Scott for the insights. For the north walls, it caused a bit headache to read through the languages in the standard, from the A12-12, it says "For Table 4B, ASHRAE CLTD values for the North exposure are adjusted.... This average is used for partitions, regardless of time of day." So the way I interpreted Table 4B is that partition wall CLTDs are derived from ASHRAE north wall values with adjustment. And the language of "Use the North CLTD values for externally shaded walls" implies that the externally shaded walls should be treated the same as partition wall in Table 4B. But the north walls, I think it makes sense to assume it to be fully shaded, and use North CLTD values (or partition wall CLTD if I read it correctly), or leave it the same as other non-directional exterior walls.
Another thing that we need to pay attention is that the Figure A12-8 group number is ASHRAE Group numbers, not the Manual J Group Numbers, and they're dark color walls. In the section A12-13 it says so and the Figure A12-12 shows the mapping. I looked up the Group B in A12-8, adjusted the color to medium (multiply 0.83), converted it to Group K in Table 4B and they're consistent. If we directly uses the A12-8 with Manual J groups then we're not doing it correctly.